src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md

```diff
@@ -156,6 +156,17 @@ By default, the Galasa Ecosystem Helm chart creates a Kubernetes Secret containi
 
 For more information on configuring Dex, see the  <a href=https://dexidp.io/docs target="_blank"> Dex documentation</a>.
 
+## Configure the 'owner' of the Galasa service
+When the Galasa service starts up, if a user is configured as one of the "owners" of the Galasa service logs in, they will have administration rights throughout the Galasa service.
+
+You need to configure the `values.yaml` file to set the `galasaOwnersLoginIds` property to refer to the login-id used by the service owner. You can set multiple owners by adding a comma-separated list of login ids.
+
+If you don't set this property, then nobody will be able to login to the service with administrator rights.
+
+Once you have logged-in as the owner, you may set the user role of other users to enable more than one administrator.
```

Comment:
I think this should be `logged in` no hyphen

Reply:
Ok. Re-worked this section to add much more explanation. Is this better ?

Reply:
re-worked section.